src/TwitchClient.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const axios = require('axios');
+const moment = require('moment');
+const TwitchEmbed = require('./embeds/TwitchEmbed');
+
+/**
+ * Manages Twitch API
+ */
+class TwitchClient {
+  /**
+   * Constructs TwitchClient
+   * @param {Discord.Client} client the discord client
+   * @param {Logger} logger the logger
+   */
+  constructor(client, logger) {
+    this.client = client;
+    this.logger = logger;
+    this.lastStartedAtTime = null;
+
+    this.client.on('ready', () => {
+      // find the channel that twitch messages will be posted to
+      this.channel = this.client.channels.get(process.env.TWITCH_NOTIFICATION_CHANNEL_ID);
+
+      // save this in closure for setInterval
+      const self = this;
+      setInterval(async () => { await self.pollTwitch(); },
+        process.env.TWITCH_POLL_INTERVAL_MS || 10000);
```

Comment:
could we elongate the minimum duration up to 1 minute (60000) so it's more in line with our other refreshes?

Reply:
Yep!